src/jquery.ime.inputmethods.js

```diff
@@ -198,6 +198,10 @@
 			name: 'फोनेटिक',
 			source: 'rules/hi/hi-phonetic.js'
 		},
+		'hoc': {
+			name: '𑢹𑣉𑣉/Ho',
+			source: 'rules/hoc/hoc-transliteration.js'
```

Comment:
I'm having problem understanding the testing part. I've written a test, how to run it?